libindi/libs/stream/streammanager.cpp

```diff
@@ -117,15 +83,15 @@ StreamManager::~StreamManager()
 
 const char * StreamManager::getDeviceName()
 {
-    return currentCCD->getDeviceName();
+    return currentDevice->getDeviceName();
 }
 
 bool StreamManager::initProperties()
 {
     /* Video Stream */
     IUFillSwitch(&StreamS[0], "STREAM_ON", "Stream On", ISS_OFF);
     IUFillSwitch(&StreamS[1], "STREAM_OFF", "Stream Off", ISS_ON);
-    IUFillSwitchVector(&StreamSP, StreamS, NARRAY(StreamS), getDeviceName(), "CCD_VIDEO_STREAM", "Video Stream",
+    IUFillSwitchVector(&StreamSP, StreamS, NARRAY(StreamS), getDeviceName(), "INDI_VIDEO_STREAM", "Video Stream",
```

Comment:
CCD_VIDEO_STREAM is an INDI standard property. For compatibility reasons, it must be used. Maybe check driver interface and set it to this and for detector define DETECTOR_VIDEO_STREAM or something like that?